### ISSUE TITLE
FirebaseAnalytics/FirebaseAnalytics.h' file not found

### ISSUE THREAD

**Issue by vishal-android-freak** [evollu/react-native-fcm#815]
**RN Version** : 0.52.0
**react-native-fcm version**: 13.3.3

```While building module 'Firebase' imported from /Users/vishal/ezyyyy/node_modules/react-native-fcm/ios/RNFIRMessaging.h:4:
In file included from <module-includes>:1:
../../../ios/Pods/Firebase/Core/Sources/Firebase.h:1:9: fatal error: 'FirebaseAnalytics/FirebaseAnalytics.h' file not found
#import <FirebaseAnalytics/FirebaseAnalytics.h>
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /Users/vishal/ezyyyy/node_modules/react-native-fcm/ios/RNFIRMessaging.m:1:
/Users/vishal/ezyyyy/node_modules/react-native-fcm/ios/RNFIRMessaging.h:4:9: fatal error: could not build module 'Firebase'
#import "Firebase.h"
 ~~~~~~~^~~~~~~~~~~~
2 errors generated.
```

My Podfile:
```
# Uncomment the next line to define a global platform for your project
platform :ios, '9.0'
use_frameworks!

target 'EZY' do
  # Uncomment the next line if you're using Swift or would like to use dynamic frameworks

  # Pods for EZY
  pod 'Moscapsule', :git => 'https://github.com/flightonary/Moscapsule.git'
  pod 'OpenSSL-Universal'
  pod 'Firebase/Core'
  pod 'Firebase/Messaging'

end

```

I read issues similar to this but they didn't help me solve the problem. I have integrated react-native-fcm in another project a few weeks back and have followed the exact steps. ```react-native-link``` doesn't work properly and hence, I have manually linked the project by following these steps

1. Adding RNFIRMessaging.xcodep... to libraries
2. The .a file to embedded frameworks in project settings.
3. Adding headers path in build settings.

Please help

I have a bridging header as I have some native code exposed to JS. Does that make a difference?

**Comment by evollu** [comment]
do `#use_frameworks!`?

**Comment by sidferreira** [comment]
Having the same issue.
I have `use_frameworks!` enabled, and seem that not enabling it cause other issues hahaha

**Comment by vishal-android-freak** [comment]
I had to manually copy files from ```ios/Pods/Frameworks``` to ```ios/Frameworks``` and it worked. Is this missing in the README, @evollu ?

**Comment by evollu** [comment]
@vishal-android-freak it shouldn't never be under `ios/Pods/Frameworks`

**Comment by evollu** [comment]
this issue should be fixed in v14.0.0
